public/js/components/card-drop-down.jsx

```diff
@@ -20,15 +20,51 @@ export default class CardDropDown extends Component {
     ).isRequired,
     cssModifier: PropTypes.string,
     onCardChange: PropTypes.func.isRequired,
+    selectedCardResource: PropTypes.string,
+    showDefaultOption: PropTypes.bool.isRequired,
+  }
+
+  static defaultProps = {
+    showDefaultOption: true,
   }
 
   constructor(props) {
     super(props);
-    this.state = {
+    var cardData;
+    // Get the data for a the default selected card.
+    if (props.selectedCardResource) {
+      cardData = this.getSelectedCard(props.selectedCardResource);
```

Comment:
this fetches data from the API, right? I don't know if the constructor is the right place since I think React creates components then caches them. Shouldn't this go in `componentWillMount()` instead?